systemverilog-plugin/third_party/yosys/README

```diff
@@ -0,0 +1,10 @@
+Files in this directory were copied from Yosys sources and slightly adapted.
```

Comment:
Might be worth-while mentioning what adaption has been done, and why we can't link against the yosys implementation.

Reply:
Added reason, changes, and exact source revision